frontend/scripts/react-components/home/slider-section/slider-section.component.jsx

```diff
@@ -130,8 +130,13 @@ class SliderSection extends React.PureComponent {
 
     return (
       <section className={cx('c-slider-section', className)}>
+        <div className="gradient-before" />
         <div
-          className={cx('row', 'slider-wrapper', { '-auto-width': slides.length < visiblePages })}
+          className={cx('row', 'slider-wrapper', {
+            '-auto-width': slides.length < visiblePages,
+            '-yellow-fade': variant === 'profiles',
+            '-white-fade': variant !== 'profiles'
```

Comment:
if we're using variants, why not set a default variant instead of assigning all to white.